build/annotate-client-locator.php

```diff
@@ -0,0 +1,28 @@
+<?php
+/**
+ * This file updates the '@method' annotations on the Aws\Sdk class.
+ */
+
+require __DIR__ . '/../vendor/autoload.php';
+
+$namespaces = array_map(function (array $manifest) {
+    return $manifest['namespace'];
+}, array_values(Aws\manifest()));
+
+sort($namespaces);
+
+$annotations = array_map(function ($namespace) {
+    return " * @method \\Aws\\{$namespace}\\{$namespace}Client"
+        . " create{$namespace}(array \$args = [])";
+}, $namespaces);
+$previousAnnotationPattern = '/^\* @method'
+    . ' \\\\Aws\\\\(?:[a-zA-Z0-9]+)\\\\(?:[a-zA-Z0-9]+)Client'
+    . ' create(?:[a-zA-Z0-9]+)\\(array \$args = \\[\\]\\)/';
+
+(new ClassAnnotationUpdater(
```

Comment:
This should be separate statements I think. First create, and then call. Spanning multiple lines in a ctor and then calling a method in the expression makes this hard to read.